app/Extensions/WoocommerceAutoCatThumbnails.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+namespace BlazeWooless\Extensions;
+
+class WoocommerceAutoCatThumbnails {
+    public static $instance = null;
+
+    public static function get_instance()
+    {
```

Comment:
@zildjianjax  formating of the functions are not correct . see brace style https://developer.wordpress.org/coding-standards/wordpress-coding-standards/php/#brace-style